source/common/http/filter_manager.cc

```diff
@@ -1197,6 +1199,7 @@ void FilterManager::encode1xxHeaders(ActiveStreamEncoderFilter* filter,
     }
   }
 
+  state_.informational_response_headers_encoded_ = true;
```

Comment:
Is it possible for the filter manager to reenter a state where this should be set back to false? E.g. maybe a first 1xx response got encoded, but a second 1xx response is not yet fully processed?

Reply:
Good catch, fixed.